bleps-macros/src/lib.rs

```diff
@@ -391,10 +406,17 @@ pub fn gatt(input: TokenStream) -> TokenStream {
                 let rfunction = format_ident!("_attr_read{}", current_handle);
                 let wfunction = format_ident!("_attr_write{}", current_handle);
                 decls.push(
-                    quote!(let mut #char_ccd_data_attr = (&mut #rfunction, &mut #wfunction);),
+                    quote!(let mut #char_ccd_data_attr = (&mut #rfunction, &mut #wfunction, ());),
                 );
 
                 let backing_data = format_ident!("_attr_data{}", current_handle);
+                let notify_cb_call = if let Some(cb) = &characteristic.notify_cb {
+                    let cb = format_ident!("{}", cb);
+                    quote!(#cb(unsafe{#backing_data[0] & 0x1 == 0x1}))
+                } else {
+                    quote!()
+                };
```

Comment:
I think this is a left-over from before "Reworked that patch so that the notification is part of the AttData trait."

Reply:
Doh!  `git add -i` fail.